include/clap/ext/draft/channel-info.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "../../plugin.h"
+#include "../../color.h"
+#include "../../string-sizes.h"
+
+static CLAP_CONSTEXPR const char CLAP_EXT_CHANNEL_INFO[] = "clap.channel-info.draft/0";
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum {
+   CLAP_CHANNEL_INFO_HAS_TRACK_NAME = (1 << 0),
+   CLAP_CHANNEL_INFO_HAS_TRACK_COLOR = (1 << 1),
+   CLAP_CHANNEL_INFO_HAS_TRACK_ID = (1 << 1),
+   CLAP_CHANNEL_INFO_HAS_TRACK_PATH = (1 << 1),
+   CLAP_CHANNEL_INFO_HAS_CHANNEL_NAME = (1 << 2),
+   CLAP_CHANNEL_INFO_HAS_CHANNEL_COLOR = (1 << 3),
+   CLAP_CHANNEL_INFO_HAS_CHANNEL_ID = (1 << 4),
+   CLAP_CHANNEL_INFO_HAS_CHANNEL_PATH = (1 << 4),
+   CLAP_CHANNEL_INFO_HAS_AUDIO_CHANNEL_COUNT = (1 << 5),
+   CLAP_CHANNEL_INFO_HAS_AUDIO_PORT_TYPE = (1 << 6),
+};
```

Comment:
Minor thing, but it's probably a good idea to have this follow the same order as the struct field.